guava/src/com/google/common/base/Ascii.java

```diff
@@ -17,6 +17,8 @@
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkNotNull;
 
+import org.checkerframework.checker.index.qual.NonNegative;
```

Comment:
As a matter of style, Google puts imports in alphabetical order, and with no blank lines (except a blank line separating `import static` from regular `import`).  So this pull request should conform to that style, here and elsewhere.